### PR TITLE
acm4471 new mce upgd disco clusters 2.8

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by berenss** [comment]
Ok this leaves many various manual steps for the MCE approach and looks OK to me.
Will this be titled "Disconnected cluster upgrades with multicluster engine" or "Disconnected cluster upgrades with manual steps" or something similar?  ie. how to carefully distinguish from the ACM topic.

**Comment by cadawson** [comment]
Thanks @berenss and @elgnay . I don't think we need to mention MCE in the title, as it will be within the MCE docs. Maybe one can be "Upgrading disconnected clusters manually" and one is "Upgrading disconnected clusters by using policies"? Or maybe "Upgrading a disconnected cluster" and "Upgrading multiple disconnected clusters by using policies"? Thoughts?

**Comment by berenss** [comment]
Cool -same page Chris.
I'm with: "Upgrading a disconnected cluster" and "Upgrading multiple disconnected clusters by using policies"

File: clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc [hunk 1]
```diff
@@ -190,216 +190,90 @@ spec:
...
+== Deploy a disconnected catalog source
+
+On the managed cluster, disable all of the default catalog sources and create a new one.
+Complete the following steps to change the default location from a connected location to your disconnected local registry: 
+
+. Create a YAML file named `source.yaml` that resembles the following content: 
 +
 [source,yaml]
 ----
 apiVersion: config.openshift.io/vi
```
Comment (elgnay) [inline clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc]: It should be `apiVersion: config.openshift.io/v1`

**Comment by elgnay** [comment]
LGTM

File: clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc (hunk 1 above)
Comment (cadawson) [reply clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc]: Fixed. Thanks!

File: clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc [hunk 2]
```diff
@@ -0,0 +1,365 @@
+[#upgrading-disconnected-cluster]
+= Upgrading a disconnected cluster
+
+You can use {cincinnati} with {mce-short} to upgrade cluster in a disconnected environment.
+
+In some cases, security concerns prevent clusters from being connected directly to the Internet. This makes it difficult to know when upgrades are available, and how to process those upgrades. Configuring {cincinnati-short} can help.
```
Comment (oafischer) [inline clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc]:
```suggestion
In some cases, security concerns prevent clusters from being connected directly to the internet. This makes it difficult to know when upgrades are available, and how to process those upgrades. Configuring {cincinnati-short} can help. 
```

**Pull request merged**